Python/psnlYeeshaPageChanges.py

```diff
@@ -131,88 +131,85 @@ def OnServerInitComplete(self):
         CurrentPage = 0
         
         AgeVault = ptAgeVault()
-        if type(AgeVault) != type(None): #is the Vault online?
```

Comment:
This can actually happen. Sorry if I accidentally flagged this one 😞

Reply:
I don't think you flagged it, I think I was overzealous. I restored the ones you commented on here and more, in files `nb01EmgrPhase0.py`, `nb01RPSGame.py`, `psnlBahroPoles.py`, `psnlYeeshaPageChanges.py`, and `xCheat.py`.

There are also instances where the `ptAgeVault()` is not originally checked for `None`, by the way, e.g. in `xCheat.py:GetChildInfo`.